### PR TITLE
fix: output target should be a dir not a file

### PR DESCRIPTION
**Pull request by nic-chen**

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
@EnableAsync
Please check other `package-xxx.sh`.

**Comment by EnableAsync** [comment]
> @EnableAsync
> Please check other `package-xxx.sh`.

OK

File: .github/workflows/package-apisix-dashboard.yml [hunk 1]
```diff
@@ -21,4 +31,24 @@ jobs:
 
       - name: run apisix-dashboard packaging test
         run: |
-          make package type=rpm app=dashboard version=2.4 checkout=v2.4
+          make package type=rpm app=dashboard version=2.7 checkout=release/2.7
+
+      - name: Run centos7 docker and mapping rmp into container
```
Comment (EnableAsync) [inline .github/workflows/package-apisix-dashboard.yml:36]: typo `rpm`.

Comment (nic-chen) [reply .github/workflows/package-apisix-dashboard.yml:36]: fixed. thanks.

**Comment by nic-chen** [comment]
I will add a test case for APISIX packaging in another PR.

**Comment by EnableAsync** [comment]
> I will add a test case for APISIX packaging in another PR.

I wrote the rpm test for apisix and I don't know if it will help you.

**Pull request merged**